tests/integration.rs

```diff
@@ -2264,6 +2279,48 @@ mod tests {
             });
         }
 
+        #[cfg(target_arch = "x86_64")]
+        #[cfg_attr(not(feature = "mmio"), test)]
+        fn test_guest_numa_nodes() {
```

Comment:
Can you use the new methodology for newly added integration tests.

Reply:
Ah yes good point :)